scripts/_helpers.py

```diff
@@ -940,7 +940,12 @@ def annuity(n, r):
 
 
 def prepare_costs(
-    cost_file: str, USD_to_EUR: float, fill_values: dict, Nyears: float | int = 1
+    cost_file: str,
+    USD_to_EUR: float,
+    fill_values: dict,
+    scenario: str,
+    market: str,
```

Comment:
Just to make sure we are on the same page: are those changes project-specific, right?
It looks we don't have `market` and `scenario` arguments in the `prepare_costs` of the upstream implementation. Is the a scenario-managment system we have implemented for the project, right?